recipes/ffn/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "ffn" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 4a79e72e06ff328e333ffe97010b1ce110bcd694fcd03be7351bf5065cd273e8 
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - decorator >=4
+    - future >=0.15
+    - matplotlib >=1
```

Should `matplotlib-base` be enough (i.e. no forced Qt dependency)?
```suggestion
    - matplotlib-base >=1
```